pkg/pillar/types/types.go

```diff
@@ -96,6 +102,12 @@ func (state SwState) String() string {
 		return "RESTARTING"
 	case PURGING:
 		return "PURGING"
+	case PENDING:
```

Comment:
Note that the ZSwState() function below also needs the new cases.

Reply:
Fixed in latest commit.